interface/patient_file/encounter/encounter_top.php

```diff
@@ -19,7 +19,7 @@
         setpid($set_pid);
     }
 
-    setencounter($_GET["set_encounter"]);
+    setencounter(attr($_GET["set_encounter"]));
```

Comment:
Doing something this far upstream will just lead to double escaping. This escaping needs to be done downstream when the $_SESSION['encounter'] and $encounter are outputted to html.